ddk-target/ddk.target

```diff
@@ -1,43 +1,45 @@
+<unit id="org.eclipse.swtbot.eclipse.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.eclipse.gef.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.ide.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.eclipse.test.junit.feature.group" version="0.0.0"/>
+<repository location="http://download.eclipse.org/technology/swtbot/releases/2.7.0/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.pde.source.feature.group" version="3.14.400.v20200604-0540"/>
+<unit id="org.eclipse.jdt.source.feature.group" version="3.18.400.v20200604-0540"/>
+<unit id="org.eclipse.platform.feature.group" version="4.16.0.v20200604-0951"/>
+<unit id="org.eclipse.platform.source.feature.group" version="4.16.0.v20200604-0951"/>
+<unit id="org.eclipse.emf.codegen.ecore.ui.feature.group" version="2.22.0.v20200324-0947"/>
+<unit id="org.eclipse.xpand.sdk.feature.group" version="2.2.0.v201605260315"/>
+<repository location="https://download.eclipse.org/releases/2020-06/202006171000/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.emf.mwe2.runtime.sdk.feature.group" version="2.12.1.v20210218-2134"/>
+<repository location="https://download.eclipse.org/releases/2021-03/202103171000/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
+<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.25.0/"/>
+</location>
+  <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+    <repository location="http://download.eclipse.org/lsp4j/updates/releases/0.10.0"/>
+    <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.10.0.v20201105-1605"/>
+  </location>
+  <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+    <repository location="https://download.eclipse.org/tools/orbit/downloads/drops/R20200529191137/repository"/>
+    <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.core.source" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
+    <unit id="org.hamcrest.library.source" version="1.3.0.v20180524-2246"/>
+    <unit id="org.mockito" version="2.23.0.v20200310-1642"/>
+    <unit id="org.mockito.source" version="2.23.0.v20200310-1642"/>
+  </location>
+</locations>
+</target>
```

Comment:
Hi Fredi,

it looks like we have here change in the end of line character(s), is this wanted or shall we revert it?

Reply:
done